### PR TITLE
added new points for tpkd day2 work

### PR DESCRIPTION
**Pull request by conroy-chee**
added new subfield points

- oil
- standby
- motor
- winding
- bearing
- inhibit

### CODE REVIEW

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 1]
```diff
@@ -136,6 +140,7 @@ descriptor:
   north: "Cardinal direction; opposite of south"
   occupancy: "State of being occupied."
   occupied: "State of being within scheduled (time of day or week) run-time parameters."
+  oil: "Oil in heater."
```
Comment (tasodorff) [inline ontology/yaml/resources/subfields/subfields.yaml:143]: Too specific to the application and references itself in its own definition; try this instead: "a viscous liquid derived from petroleum, especially for use as a fuel or lubricant"

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 2]
```diff
@@ -42,6 +43,7 @@ component:
   heater: "Component which provides heat to media."
   humidifier: "Component which humidifies."
   lamp: "The component of a lighting device that emits the light."
+  motor: "The component of a compressor."
```
Comment (tasodorff) [inline ontology/yaml/resources/subfields/subfields.yaml:46]: Motor: a machine, especially one powered by electricity or internal combustion, that supplies motive power for a vehicle or for some other device with moving parts.

**Review by tasodorff — CHANGES_REQUESTED** [review]
Minor changes to definitions requested.

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 2 above)
Comment (conroy-chee) [reply ontology/yaml/resources/subfields/subfields.yaml]: the description has been changed

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 1 above)
Comment (conroy-chee) [reply ontology/yaml/resources/subfields/subfields.yaml]: the description for oil has been changed

**Comment by conroy-chee** [comment]
hi @tasodorff , could you help to review the amended descriptions that I have changed?

**Review by tasodorff — APPROVED** [review]

**Comment by conroy-chee** [comment]
hi @tasodorff , could you help to merge to maste since you approved thisr? Thank you.

**Pull request merged**